cas-server-core-api/src/main/java/org/jasig/cas/ticket/TicketGrantingTicket.java

```diff
@@ -66,11 +66,12 @@
      * @param service The service for which we are granting a ticket
      * @param expirationPolicy the expiration policy.
      * @param credentialsProvided if the credentials are provided.
+     * @param onlyTrackMostRecentSession track the most recent session by keeping the latest service ticket
      * @return the service ticket granted to a specific service for the
      * principal of the TicketGrantingTicket
      */
     ServiceTicket grantServiceTicket(String id, Service service,
-        ExpirationPolicy expirationPolicy, boolean credentialsProvided);
+        ExpirationPolicy expirationPolicy, boolean credentialsProvided, boolean onlyTrackMostRecentSession);
```

Comment:
Glad we see things the same way now ;-)

Will update PRs and use constructor to avoid any breaking change.